source/extensions/transport_sockets/tls/stats.cc

```diff
@@ -14,6 +14,11 @@ SslStats generateSslStats(Stats::Scope& store) {
                         POOL_HISTOGRAM_PREFIX(store, prefix))};
 }
 
+CertStats generateCertStats(Stats::Scope& scope, std::string cert_name) {
+  std::string prefix_cert_name("ssl." + cert_name + ".");
```

Comment:
```suggestion
  std::string prefix_cert_name(absl::StrCat("ssl.", cert_name, "."));
```

Reply:
I updated naming to `ssl.certificate.<cert_name>.`